elements/lisk-p2p/test/unit/peer_pool.ts

```diff
@@ -386,60 +921,6 @@ describe('peerPool', () => {
 		});
 	});
 
-	describe('#request', () => {
-		describe('when no peers are found', () => {
-			let caughtError: Error;
-			beforeEach(async () => {
-				(peerPool as any)._peerSelectForRequest = sandbox
-					.stub()
-					.returns([] as ReadonlyArray<P2PPeerInfo>);
-				try {
-					await peerPool.request({ procedure: 'proc', data: 123 });
-				} catch (err) {
-					caughtError = err;
-				}
-			});
-
-			it('should call _peerSelectForRequest with all the necessary options', async () => {
-				expect((peerPool as any)._peerSelectForRequest).to.be.calledWith({
-					peers: [],
-					nodeInfo: peerPool.nodeInfo,
-					peerLimit: 1,
-					requestPacket: { procedure: 'proc', data: 123 },
-				});
-			});
-
-			it('should throw an error', async () => {
-				expect(caughtError).to.not.be.null;
-				expect(caughtError)
-					.to.have.property('name')
-					.which.equals('RequestFailError');
-			});
-		});
-
-		describe('when peers are found', () => {
-			it('should not throw an error');
-		});
-	});
-
-	describe('#send', () => {
-		beforeEach(async () => {
-			(peerPool['_peerSelectForSend'] as any) = sandbox
-				.stub()
-				.returns([] as ReadonlyArray<P2PPeerInfo>);
-			peerPool.send({ event: 'foo', data: 123 });
-		});
-
-		it('should call _peerSelectForSend with all the necessary options', async () => {
-			expect(peerPool['_peerSelectForSend']).to.be.calledWith({
-				peers: [],
-				nodeInfo: peerPool.nodeInfo,
-				peerLimit: DEFAULT_SEND_PEER_LIMIT,
-				messagePacket: { event: 'foo', data: 123 },
-			});
-		});
-	});
-
 	describe('#triggerNewConnections', () => {
```

Comment:
Could you order these describe sections as they exist in `peep_pool.ts` file? Please, add skeletons if needed for those methods not addressed in this issue.